### DISCUSSION TITLE
What version of Bootstrap does IHP use?

### DISCUSSION THREAD

**Discussion by neongreen** [digitallyinduced/ihp#1045] [Q&A] [answered]
I'm not sure which version of Bootstrap does IHP use. It looks like it's even older than 3.3, because `.button-default` exists in Bootstrap 3.3 but not in IHP (see https://getbootstrap.com/docs/3.3/css/#buttons).

Ideally I'd like to use Bootstrap 4.

Is it possible to change the Bootstrap version?

**Comment by mpscholten** [comment, marked as answer]
IHP is currently using bootstrap 4 👍 You're looking for the `btn-primary` class :)

**Reply by neongreen** [reply]
Oh alright. Which version specifically? 4.6?

**Reply by mpscholten** [reply]
Currently `v4.5.0` :)

**Reply by neongreen** [reply]
Gotcha.
